TestRunner.cfm

```diff
@@ -0,0 +1,44 @@
+<cfsilent>
+<!---
+ Runs TestBox test and outputs in Codewars format.
+ To use, run:
+     box -clishellpath=/path/to/TestRunner.cfm
+ --->
+<cfscript>
+	// Bootstrap TestBox framework
+	currentDir = getDirectoryFrompath( getCurrentTemplatePath());
+	createMapping( '/testbox', currentDir & 'testbox' );
+	createMapping( '/codewarsRoot', currentDir );
+
+	// Create TestBox and run the tests
+	testData = new testbox.system.TestBox( options={ coverage : { enabled : false } } )
+		.runRaw(
+			directory = {
+				// Find all CFCs in this directory that ends with Test.
+				mapping : '/codewarsRoot',
+				recurse : false,
+				filter = function( path ){
+					return path.reFindNoCase( "Test\.cfc$" );
+				}
+			}
+		)
+		.getMemento( includeDebugBuffer=true );
+
+	new CodewarsReporter().render( testData );
+
+	// Set exit code to 1 on failure
+	if ( testData.totalFail || testData.totalError ) {
+		return 1;
```

Comment:
Does this set the exit code properly? It doesn't seem like it does when I try.

Reply:
Oh good call, that was another thing the CommandBox bootstrap provided.  Easy fix, let me add that in.

Reply:
@kazk Ok, updated.  Please give that another try.